### PR TITLE
#170869002 admin issue book

### PR DESCRIPTION
**Pull request by blaise82**
#### What does this PR do?
- issue book endpoint

#### Description of Task to be completed?
- created issue controller
- created issue routes
- changed issued table to issues
- wrote tests for issue controller

#### How should this be manually tested?
- clone this repository
- checkout to _ft-issue-book-170869002_ branch
- run `npm run install`
- run `npm run createTables`
- run npm start

`route - localhost:8080/api/v1/issue/add`
#### Screenshots (if appropriate)
Screen Shot 2020-01-29 at 18 01 59

[finishes #170869002]

### CODE REVIEW

**Comment by izzett222** [comment]
it looks great, what's left is the isAdmin auth middleware

**Review by RusimbiPatrick — COMMENTED** [review]
Remove the seeds from coverage

**Review by RusimbiPatrick — APPROVED** [review]
Looks good

File: package.json [hunk 1]
```diff
@@ -54,14 +54,17 @@
...
     "sequelize": "^5.21.3",
     "uuid": "^3.4.0"
   },
   "nyc": {
     "include": "server",
     "exclude": [
-      "build", "**/index.js"
+      "build",
+      "**/index.js",
+      "**/db"
```
Comment (AbdoulNuru) [inline package.json:67]: Why did you exclude the db folder in nyc

Comment (RusimbiPatrick) [reply package.json:67]: to stop the previously failing coverage caused by seeders. After all, we don't need to test anything in the db folder. Do we?

**Review by izzett222 — APPROVED** [review]

**Review by AbdoulNuru — APPROVED** [review]

**Pull request merged**